ydb/core/tablet_flat/flat_stat_table.h

```diff
@@ -140,40 +140,38 @@ class TKeyAccessSample {
         ui64 idx = TotalCount;
         ++TotalCount;
         if (idx >= SampleCount) {
-            idx = RandomNumber<ui64>(TotalCount) ;
+            idx = RandomNumber<ui64>(TotalCount);
         }
 
         if (idx >= SampleCount) {
             return;
         }
 
-        TSerializedCellVec saved(TSerializedCellVec::Serialize(key));
+        TString serializedKey = TSerializedCellVec::Serialize(key);
 
-        auto it = KeyRefCount.find(saved.GetBuffer());
-        if (it != KeyRefCount.end()) {
+        auto [it, inserted] = KeyRefCount.emplace(serializedKey, 1);
```

Comment:
I thinks this should be replaced with `++KeyRefCount[serializedKey]`. The `it` iterator is not used for anything else anyway.

Reply:
Updated.